docs/adrs/0003-support-aggregations-on-publisher-name.md

```diff
@@ -0,0 +1,183 @@
+- logic for extracting dates and locations from publisher information could be shared across all Transmogrifier sources
+  - e.g. it could be an automatically applied, secondary step after the `PublicationInfo` objects are created, pulling from `PublicationInfo.date` and `Publication.location`
+  - allows for more thorough date parsing for `Dates` entries, without losing meaningful strings from the source record that can remain in the `PublicationInfo` object
+
+Example record:
+```json
+{
+  "publication_info": [
+    {
+      "name": "Great Writings",
+      "date": "1930",
+      "location": "Bend, OR"
+    },
+    {
+      "name": "Amazon Reprints",
+      "date": "2020",
+      "location": "Seattle, WA"
+    },
+    {
+      "name": "Ebooks Inc.",
+      "date": "Circa 2023?"
+    }
+  ],
+  "dates": [
+    {
+      "kind": "Published",
+      "value": "1930"
+    },
+    {
+      "kind": "Published",
+      "value": "2020"
+    }
+  ],
+  "locations": [
+    {
+      "kind": "Published",
+      "value": "Bend, OR"
+    },
+    {
+      "kind": "Published",
+      "value": "Seattle, WA"
+    }
+  ]
+}
+```
+- note that bad or missing data from publisher "Ebooks Inc." is skipped for `dates` and `locations` extraction
+
+This avoids some subtle but potentially confusing scenarios:
+
+- **Option 1**: a user clicks date facet "1910" in search UI but does not see "1910" under "Dates" in the item page
+  - **Reason**: the UI item page didn't know it should reach into `PublicationInfo` objects for dates to show under "Dates", as this was custom logic applied to GraphQL aggregations and search facets
+  - **Option 3 fix**: GraphQL, UI search, and UI item pages all pull publishers from publishers, dates from dates, locations from locations, etc., no logic required
+
+- **Option 2**: a user is viewing an item page for the geospatial record "Fires in 1999 Dataset" but sees a strange "2020" under the "Dates" section
+  - **Reason**: the item page "Publisher" section only shows "GIS Pro Inc.", as "2020" was decoupled from the publisher (`Date` object would contain `@kind=Published` qualifier, but we'd need to then include that in the item page)
+  - **Option 3 fix**: the "Publisher" section in the item page clearly also shows "2020", because still part of a complex object, thereby contextualizing that date
+
+Option 3 achieves data where and when needed, and with the appropriate amount of context, by _extracting and duplicating_ some data like dates and locations:  
+
+- a user wants details about a publisher, look at full and complex `PublicationInfo` object in the record
+- the API or UI wants to pull all meaningful dates or locations from a record, look to the `Dates` or `Locations` fields
+
+In either situation, no additional logic, mapping, or documentation is needed. 
+
+## Decision
+
+Proceed with Option 3:
+
+- create new, top level, multivalued object field `PublicationInfo` with properties `[name, date, location]`
```

Comment:
Can this be `PublicationInformation`? We aren't abbreviating any of the other `transmogrifier` field names or classes

Reply:
I'm a little fuzzy on GraphQL deprecation, but perhaps that is contributing to this variation on `PublicationInformation` which already existed?

Is it correct to assume @JPrevost, @jazairi, @matt-bernhardt, that if we changed the field type of the **pre-existing** `PublicationInformation` field, that doesn't give us the ability to really deprecate it?  Because from GraphQL's POV the field name didn't change, so no ability to say, "This field is deprecated, use XYZ..."?

If so, @ehanson8 and @jazairi (as thumbs-uppers for this suggestion), what would you think about `Publishers` as the complex object field name?  My assumption is that "Information" or "Info" was originally proposed in the event we don't have a name, where just a date or location feels a little odd as a "Publisher".  But, that's generally only the case for copyright dates, and we could elect to skip a `Publisher` entry if only that copyright date, and instead just write that directly to `Dates`.

All considered, I suppose my vote would be `Publishers` as the top level field for complex objects, thereby matching other field name patterns like `Contributors`.

Reply:
I'm fine with whatever so long as it's not abbreviated 🙂

Reply:
Oh, yes, good point about the name conflict. I'd be happy with `Publishers` as an alternative.

Reply:
@ehanson8, @jazairi - field name updated from `PublicationInfo` to `Publishers` in this most commit: https://github.com/MITLibraries/transmogrifier/pull/132/commits/9bb06e567a9195c4cdea92c48d6693a17169cf27